internal/controllers/vehicle_subscription_controller.go

```diff
@@ -231,8 +300,104 @@ func (v *VehicleSubscriptionController) SubscribeAllVehiclesToWebhook(c *fiber.C
 		}
 		successCount++
 	}
+	return c.Status(http.StatusCreated).JSON(fiber.Map{
+		"message": fmt.Sprintf("Successfully subscribed %d vehicles to the webhook", successCount),
+	})
+}
+
+// SubscribeMultipleVehiclesToWebhook godoc
+// @Summary      Subscribe multiple vehicles to a webhook
+// @Description  Accepts a comma-separated list of vehicle token IDs and subscribes each one to the specified webhook event.
+// @Tags         Vehicle Subscriptions
+// @Produce      json
+// @Param        eventID path string true "Event ID"
+// @Param        vehicleTokenIDs query string true "Comma-separated list of vehicle token IDs"
+// @Param        Authorization header string true "Bearer token"
+// @Success      201 {object} map[string]string "Successfully subscribed X vehicles to the webhook"
+// @Failure      400 {object} map[string]string "Event ID and vehicle token IDs are required"
+// @Failure      401 {object} map[string]string "Unauthorized"
+// @Failure      500 {object} map[string]string "Internal server error"
+// @Router       /subscriptions/{eventID} [post]
+func (v *VehicleSubscriptionController) SubscribeMultipleVehiclesToWebhook(c *fiber.Ctx) error {
+	eventID := c.Params("eventID")
+	if eventID == "" {
+		return c.Status(http.StatusBadRequest).JSON(fiber.Map{"error": "Event ID is required"})
+	}
+
+	vehicleIDsParam := c.Query("vehicleTokenIDs")
```

Comment:
We should put these id's in the POST body as a JSON array instead of in the querystring. Typically you want the payload that's being used to modify something as part of a POST request to go in the body.  It should also make for a cleaner implementation vs. comma delimited. URL's have a max length of 2048 characters too for if somebody submits many ID's.